### PR TITLE
Add Sony ILCE-7M4 camera support

### LINKED ISSUE #328: Add support for Sony A7 IV
Hi Roman,

I've just uploaded RAW files for the A7M4 to raw.pixls.us. Let me know if you need any additional pictures.

Thanks!

### PR DESCRIPTION
**Pull request by cryptomilk**
Fixes #328

### CODE REVIEW

File: data/cameras.xml [hunk 1]
```diff
@@ -7930,6 +7930,17 @@
...
 	</Camera>
+	<Camera make="SONY" model="ILCE-7M4">
+		<ID make="Sony" model="ILCE-7M4">Sony ILCE-7M4</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
```
Comment (kmilos) [inline data/cameras.xml:7941]: There's definitely a need to remove some pixels on the right side, the question is - how many exactly?

**Comment by LebedevRI** [comment]
https://raw.pixls.us/ says
```
We are looking for shots that are:
* Image in focus and properly exposed
```
the RPU samples are not. I can not guess the right crop from them.
They really need to be landscape shots, bright, with sharp patterns all across.

**Comment by LebedevRI** [comment]
I just can't tell what the right crop here is.
It seems like at least -8, so I've gone ahead with -10, but it might be too conservative still. (could be up to -30).

**Comment by kmilos** [comment]
-10 was my best guess as well for the RPU sample...

**Pull request merged**